bookstore/src/App.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+import { Routes, Route } from 'react-router-dom';
+import Header from './components/Header'
+import BooksList from './components/BooksList';
+import Categories from './components/Categories';
+
+/*eslint-disable no-useless-constructor*/ 
+
+
+class App extends React.Component {
+  constructor(props){
+    super(props)
+  }
+
+  render(){
+    return(
+      <div>
+        <h1> Book store app</h1>
+        <Header/>
+        <Routes>
+          <Route path="/BooksList" element={<BooksList />} />
```

Comment:
- [ ] Kindly note that the BookList component should be rendered at the root("/") of the project as the requirement says. This will make it your default component when the app loads. Hint :point_down: 

```jsx
<Route exact path="/" element={<BookList />}  />
````